autopts/pybtp/btp/bap.py

```diff
@@ -130,7 +130,7 @@ def bap_send(ase_id, data_ba, bd_addr_type=None, bd_addr=None):
 def bap_broadcast_source_setup(
         streams_per_subgroup, subgroups, coding_format, vid, cid,
         codec_ltvs, sdu_interval, framing, max_sdu, retransmission_number,
-        max_transport_latency, presentation_delay):
+        max_transport_latency, presentation_delay, broadcast_id=0x123456):
```

Comment:
```suggestion
        max_transport_latency, presentation_delay, broadcast_id):
```

I would suggest not to provide any default values for the `broadcast_id`

Reply:
But then we would have to add them in every function that calls this, meaning the same value would be added regardless. So it was a means of reducing the breaking changes

Reply:
> So it was a means of reducing the breaking changes

If you are updating all calls to this, there are no breaking changes :) 

I think `bap_broadcast_source_setup` was originally defined as it is because Zephyr would generate the broadcast ID automatically. Since that is no longer the case, someone has to assign a broadcast ID. 

You are right that changing it so that it doesn't default doesn't really change anything if everyone would just call this with the same value, but now that broadcast_id is an IXIT value (at least for some profiles), I think it makes sense to set this explicitly, and to the broadcast ID set in the IXIT value. 

If PTS starts using the IXIT value for more tests/profiles, then it's easier to maintain that way